File: src/main/java/net/lomeli/lomlib/client/models/IColorProvider.java
package net.lomeli.lomlib.client.models;

import net.minecraft.client.renderer.color.IItemColor;

import net.minecraftforge.fml.relauncher.Side;
import net.minecraftforge.fml.relauncher.SideOnly;

public interface IColorProvider {
    @SideOnly(Side.CLIENT)
    IItemColor getColor();
}

File: src/main/kotlin/net/lomeli/lomlib/client/models/IColorProvider.kt
package net.lomeli.lomlib.client.models

import net.minecraft.client.renderer.color.IItemColor
import net.minecraftforge.fml.relauncher.Side
import net.minecraftforge.fml.relauncher.SideOnly

interface IColorProvider {
    @SideOnly(Side.CLIENT) fun getColor() : IItemColor
}